src/genie/libs/parser/sros/show_router_arp_dynamic.py

```diff
@@ -0,0 +1,71 @@
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+import re
+
+# ======================================================
+# Schema for 'show router arp dynamic'
+# ======================================================
+
+class ShowRouterArpDynamicSchema(MetaParser):
+	"""Schema for show router arp dynamic"""
+	schema = {
+		'router': {
+			Any(): {
+				"entries" : int,
+				'interfaces': {
+					Any(): {
+						'ip_add': str,
+						'mac_add': str,
+						'expiry': str,
+						'type':str
+					}
+				}
+			}
+		}
+	}
```

Comment:
I think it will probably be better to have `ip_address` be the key that sets table entries apart. I found documentation for similar commands and from what I could tell, it's possible for multiple table entries to have the same interface name.

What are your thoughts about a schema like this?
```
class ShowRouterArpDynamicSchema(MetaParser):
	"""Schema for show router arp dynamic"""
	schema = {
		'router': {
			Any(): {
				"num_entries" : int,
				'ip_address': {
					Any(): {
						'interface': str,
						'mac_add': str,
						'expiry': str,
						'type':str
					}
				}
			}
		}
	}
```

Reply:
Yes, sounds good to me, I will rewrite the schema on the next commit, thanks!

Reply:
Confirmed the change